### ISSUE TITLE
[Woo POS] Ensure POS works with other languages and RTL text

### ISSUE THREAD

**Issue by iamgabrielma** [woocommerce/woocommerce-ios#12707]
Despite the initial POC being US only, the store could use different languages, text could be presented in LTR, or should be build using accessibility features.

Let's assure the initial wireframe takes these into account and are unit tested as needed, so then we can apply them organically moving forward.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by staskus** [comment]
Accessibility support also depends on which will be addressed in other tasks:
- Dark Mode
- Scaling fonts
- Adapting images (SF fonts for images)

**Comment by joshheald** [comment]
> Adapting images (SF fonts for images)

Done in #13649

**Comment by staskus** [comment]
> Scaling fonts

Done in:

- https://github.com/woocommerce/woocommerce-ios/pull/13611
- https://github.com/woocommerce/woocommerce-ios/pull/13655

**Comment by joshheald** [comment]
Dark mode is covered in #13471 

This ticket is now just about locale testing, so I'm changing the name to reflect that. Accessibility is all covered elsewhere.

**Comment by staskus** [comment]
@joshheald , @iamgabrielma i18n is definitely what we need to support properly but I propose we move it to "Consider after POC". There may be not much work on this but I imagine it will again require going through every single view and making some design decisions which I think could be done after M2 deadline. Wdyt?

**Comment by iamgabrielma** [comment]
Agree, we have the default a18n and RLT out-of-the-box support, but if we want to fine-tune it and test all cases I think this can be done post-M2 during the testing phase.

**Comment by malinajirka** [comment]
Let's aim to focus on accessibility as part of project development and report specific bugs if we find any afterwards.
